modules/shared/logs/source-watcher.go

```diff
@@ -135,8 +136,14 @@ func NewSourceWatcher(cm k8shelpers.ConnectionManager, sourcePaths []string, opt
 			continue
 		}
 
+		// Show logs from all containers in a pod
+		if sw.allContainers {
+			pp.ContainerName = "*"
```

Comment:
Let's make `--all-containers` the default but let users override it with an explicit name. For example users should be able to do this:

```console
kubetail logs deployment/frontend/nginx deployment/backend --all-containers
```

Reply:
updated this.

Reply:
🔥